assets/components/headers/simpleHeader/simpleHeader.jsx

```diff
@@ -2,37 +2,133 @@
 
 // ----- Imports ----- //
 
-import React, { type Node } from 'react';
+import React, { Component, type Node } from 'react';
 
 import { type Option } from 'helpers/types/option';
+import { classNameWithModifiers } from 'helpers/utilities';
+import { onElementResize, type ElementResizer } from 'helpers/layout';
 import SvgGuardianLogo from 'components/svgs/guardianLogo';
-import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
+
+import { links } from './links';
 
 import './simpleHeader.scss';
 
 export type PropTypes = {|
   utility: Option<Node>,
+  displayNavigation: boolean,
+|};
+export type State = {|
+  fitsLinksInOneRow: boolean,
 |};
 
-// ----- Component ----- //
 
-export default function SimpleHeader({ utility }: PropTypes) {
+// ----- Metrics ----- //
 
+const willMenuFitInOneRow = ({ menuRef, logoRef, containerRef }) => {
+  const [logoWidth, menuWidth, containerWidth] = [
+    logoRef.getBoundingClientRect().width,
+    menuRef.getBoundingClientRect().width,
+    containerRef.getBoundingClientRect().width,
+  ];
   return (
-    <header className="component-simple-header">
-      <LeftMarginSection>
-        <div className="component-simple-header__content">
-          <div className="component-simple-header__utility">{utility}</div>
-          <a className="component-simple-header__link" href="https://www.theguardian.com">
-            <div className="accessibility-hint">The guardian logo</div>
-            <SvgGuardianLogo />
-          </a>
-        </div>
-      </LeftMarginSection>
-    </header>
+    containerWidth - logoWidth - menuWidth > 0
   );
+};
+
+
+// ----- Component ----- //
+
+type HeaderTopNavigationPropTypes = {|
+  utility: Option<Node>,
+  getLogoRef: (?Element) => void
+|};
 
+const HeaderTopNavigation = ({ getLogoRef, utility }: HeaderTopNavigationPropTypes) => (
+  <div className="component-simple-header-topnav">
+    <div className="component-simple-header-topnav__utility">{utility}</div>
+    <div className="component-simple-header-topnav-logo" ref={getLogoRef}>
+      <a className="component-simple-header-topnav-logo__graun" href="https://www.theguardian.com">
+        <div className="accessibility-hint">The Guardian logo</div>
+        <SvgGuardianLogo />
+      </a>
+    </div>
+  </div>
+);
+
+type HeaderBottomNavigationPropTypes = {|
+  getMenuRef: (?Element) => void
+|};
+
+const HeaderBottomNavigation = ({ getMenuRef }: HeaderBottomNavigationPropTypes) => (
+  <nav className="component-simple-header-bottomnav">
+    <ul className="component-simple-header-bottomnav__ul" ref={getMenuRef}>
+      {links.map(({ href, text }) => (
```

Comment:
it could be nice if the header accepted links as an argument rather than hardcoding the dependency here - that way the header could easy be used with different nav bars